modules/system/models/requestlog/columns.yaml

```diff
@@ -5,8 +5,7 @@
 columns:
     status_code:
         label: system::lang.request_log.status_code
-        searchable: yes
```

Comment:
Why did you remove these properties?

Reply:
Because status_code is always 404

Reply:
Just leave it as it is and set invisible to true.